### ISSUE TITLE
Arithmetic Simplification Ignored Shift Tests

### ISSUE THREAD

**Issue by dngreenst** [SpartanRefactoring/Spartanizer#415]
The arithmetic (#92) tests with shift are ignored, with the message "Do not compute shifts, they have a reason". Not sure what to make of it. Moving them to a separate test class.

**Comment by yossigil** [comment]
This is deliberately unactive. Mark the test class (not methods) `@Ignore`

**Comment by yossigil** [comment]
The GUI should make it possible to enable these, by default these should be off. Testing should be regardless.

**Comment by OriRoth** [comment]
@yossigil it is currently possible only by category. We may however create a category for `experimental-tippers`, deactivated by default, if you are interested.

**Comment by yossigil** [comment]
No, we should have a full tree, just like the warnings of Eclipse dialog box. It is heavy but doable, and the experience gained? invaluable!

**Comment by yossigil** [comment]
This is the tipper you asked for: it should be off by default, but users may install it
